public/scripts/response.js

```diff
@@ -0,0 +1,16 @@
+function submitData() {
+  $.ajax({
```

Comment:
Can we achieve this without using jQuery?

Reply:
Yes. We can use the fetch API of JS or Axios